.github/workflows/stable-release.yml

```diff
@@ -0,0 +1,35 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    branches:
+      - 'main'
+      - 'v*'
+
+    paths: 
+      - 'STABLE-RELEASE'
+
+name: KubeArmor stable release 
+
+jobs:
+  build:
+    name: Create KubeArmor Release - 18.04
+    runs-on: ubuntu-18.04
+    timeout-minutes: 20
+    steps:
+      - name: Checkout KubeArmor code
+        uses: actions/checkout@v2
+
+      - name: Pull Docker Hub image
+        run: |
+            value=`cat STABLE-RELEASE`
+            docker pull docker.io/kubearmor/kubearmor:$value
+            docker image tag kubearmor/kubearmor:$value kubearmor/kubearmor:stable
+
+      - name: Login to Docker Hub
```

Comment:
Shouldn't this step be handled before "Pull Docker Hub image" step? Since the earlier step requires docker authentication.

Reply:
Do you mean the docker pull? Docker pull doesn't require login

Reply:
I mean `docker image tag` would require authentication.